vhd_format/jbuild

```diff
@@ -3,6 +3,7 @@
   (public_name vhd-format)
   (libraries
    (cstruct
+    integers
```

Comment:
We also need to add this dependency to `vhd-format.opam` to make the Travis build work.

Reply:
Hi, Gabor, thanks for the review.
I added `integers` to `vhd-format.opam` and `vhd-format-lwt.opam`, but seems it still failed because of the error below:
```# File "vhd_format_lwt/jbuild", line 7, characters 4-18:```
```# Error: Library "lwt.preemptive" not found.```
Is the library `lwt.preemptive` not available? Thanks. @gaborigloi

Reply:
Looks like @mseri fixed this in https://github.com/mirage/ocaml-vhd/pull/52, you can just cherry-pick that change :)

Reply:
But seems #52 is also CI failed because of : 

### stderr ###
# File "cli/jbuild", line 3, characters 12-26:
# Warning: This field is useless without a (public_name ...) field.
# Error: Library "lwt.ppx" not found.
# -> required by (pps (lwt.ppx ocaml-migrate-parsetree.driver-main))
# Hint: try: jbuilder external-lib-deps --missing -p vhd-format-lwt @runtest

Reply:
We need the `lwt_ppx` opam package for that.

Reply:
Apparently this is due to the latest lwt changes: https://discuss.ocaml.org/t/lwt-4-0-0-a-major-cleanup-of-the-repo/1838

Reply:
Yes, I try to fix CI by modify opam and jbuild file,  now vhd-format and vhd-format-lwt ok but got error below, lt's related to after cherry-pick #52.

File "disk/disk.ml", line 41, characters 35-38:
 Error: This expression has type string but an expression was expected of type
          bytes

Not related to this PR but trying to fix this lately. @gaborigloi @robhoes